.env

```diff
@@ -1,5 +1,8 @@
+# ⚠️ IMPORTANT!! ⚠️
+# When adding a new env var, don't forget to map it to `.github/workflows/vercel.yml`
+# Otherwise Vercel builds will not pick it up from Github secrets
+
```

Comment:
❤️ nice

Could you add a TODO somewhere

`// Extract envs from README using sed command (so this warning can be removed)`